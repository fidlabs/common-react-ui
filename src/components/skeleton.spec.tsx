import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { Skeleton } from "./skeleton";

describe("Skeleton component", () => {
  it("renders correctly", () => {
    const { container } = render(<Skeleton />);
    expect(container).toMatchSnapshot();
  });
});
