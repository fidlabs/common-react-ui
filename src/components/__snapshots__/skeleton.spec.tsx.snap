// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Skeleton component > renders correctly 1`] = `
<div>
  <div
    class="animate-pulse rounded-md bg-gray-300"
  />
</div>
`;
